src/commercial.ts

```diff
@@ -21,16 +34,9 @@ const chooseAdvertisingTag = async () => {
 			'./init/consentless'
 		).then(({ bootConsentless }) => bootConsentless(consentState));
 	} else {
-		bootCommercialWhenReady();
+		void import(
+			/* webpackChunkName: "consented" */
```

Comment:
We will be monitoring our commercial metrics of course to do pre/post analysis, an ab test is likely to be produce clearer results although with the complexity of serving different bundles?

Reply:
Okay great as long we are tracking the impact to commercial-start for consented. It will be interesting to see there is any impact to mobile devices which are usually more network constrained.